mappings/net/minecraft/client/option/GameOptions.mapping

```diff
@@ -109,6 +109,7 @@ CLASS net/minecraft/class_315 net/minecraft/client/option/GameOptions
 	FIELD field_26845 keySocialInteractions Lnet/minecraft/class_304;
 	FIELD field_26926 hideMatchedNames Z
 	FIELD field_28777 hideBundleTutorial Z
+	FIELD field_32156 monochromeBackground Z
```

Comment:
```suggestion
	FIELD field_32156 monochromeLogo Z
```

Something that might be useful in a later pull request is to add Javadoc showing how these options are saved. For example, this one is `darkMojangStudiosBackground`.